simulator-spring-boot/src/main/java/org/citrusframework/simulator/service/TestResultQueryService.java

```diff
@@ -118,8 +118,8 @@ protected Specification<TestResult> createSpecification(TestResultCriteria crite
             if (criteria.getErrorMessage() != null) {
                 specification = specification.and(buildStringSpecification(criteria.getErrorMessage(), TestResult_.errorMessage));
             }
-            if (criteria.getFailureStack() != null) {
-                specification = specification.and(buildStringSpecification(criteria.getFailureStack(), TestResult_.failureStack));
+            if (criteria.getStackTrace() != null) {
```

Comment:
`criteria.getStackTrace()` and all the other getters are not nullable 😇

Reply:
corrected in 309dc97e33e41a78e0a7c5c9de63f39c6e0f1639.